narwhals/_ibis/dataframe.py

```diff
@@ -44,6 +44,10 @@ def map_ibis_dtype_to_narwhals_dtype(
         return dtypes.Date()
     if ibis_dtype.is_timestamp():
         return dtypes.Datetime()
+    if ibis_dtype.is_array():
+        return dtypes.List()
```

Comment:
looks like they don't support Array, and what they call Array corresponds to Polars' List

I've repored this to them